src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -67,4 +67,9 @@ public CommandResult execute() {
         }
     }
 
+    @Override
+    public boolean isMutating(){
```

Comment:
Coding standard violation. Remember to include a space before the opening brace.